src/clusterfuzz/_internal/platforms/android/flash.py

```diff
@@ -121,10 +122,9 @@ def flash_to_latest_build_if_needed():
   branch = environment.get_value('BUILD_BRANCH')
   target = environment.get_value('BUILD_TARGET')
   if not target:
-    # We default to userdebug configuration.
     build_params = settings.get_build_parameters()
     if build_params:
-      target = build_params.get('target') + '-userdebug'
```

Comment:
This issue only affects  cuttlefish right? Can you add a check for cuttlefish?

Reply:
Added. This helps with clarity, but Cuttlefish emulators are the only users of this code. Flashing is disabled (through ClusterFuzz, we handle it in the lab) for physical devices.

Reply:
I still think there's a change here that affects Chrome.
Could you change it to:
```
    if build_params:
      if environment.is_android_cuttlefish():
        target = build_params.get('target') + FLASH_DEFAULT_BUILD_TARGET
      else:
        target = build_params.get('target') + '-userdebug'
```

Reply:
Done.